go/vt/schemamanager/tablet_executor.go

```diff
@@ -267,12 +263,11 @@ func (exec *TabletExecutor) executeSQL(ctx context.Context, sql string, execResu
 			execResult.ExecutorErr = err.Error()
 			return err
 		}
-		if exec.ddlStrategySetting.IsSkipTopo() {
-			exec.executeOnAllTablets(ctx, execResult, onlineDDL.SQL, true)
-			exec.wr.Logger().Printf("%s\n", onlineDDL.UUID)
-		} else {
-			exec.executeOnlineDDL(ctx, execResult, onlineDDL)
-		}
+		exec.executeOnAllTablets(ctx, execResult, onlineDDL.SQL, true)
+		exec.wr.Logger().Printf("%s\n", onlineDDL.UUID)
+		return nil
+	case *sqlparser.AlterMigration:
+		exec.executeOnAllTablets(ctx, execResult, sql, true)
```

Comment:
Here is the actual change in this PR: direct `AlterMigration` statements to the tablets via `QueryService` (that's what the last `true` argument indicates).